config/default/config_ignore.settings.yml

```diff
@@ -5,5 +5,7 @@ ignored_config_entities:
   6: shield.settings
   8: 'migrate_plus.migration.*'
```

Comment:
Is there a way that we can only ignore sample migrations? I would say there is still a legitimate use case for creating migration templates as part of a project.

Reply:
@Benjen I looked in to it and it seems like we cannot differ between the bundled migrations and new ones. We will have to clean that up manually if need be.